src/client/app/strings/index.ts

```diff
@@ -165,6 +165,7 @@ export interface ClientStrings {
     aboutStorage: string;
     aboutSnippets: string;
     aboutIntellisense: string;
+    aboutCurrentEnvironment: string;
```

Comment:
Note, with the changes that I just merged from Mourad (German localization), your code will fail once you merge in his changes.  Follow the pattern that I did for `snippetInfoDialogTitle: englishSubstitutesForNotYetTranslated.snippetInfoDialogTitle,` in German.ts in the client strings.